cilium/cmd/debuginfo.go

```diff
@@ -121,3 +136,20 @@ func printMD(w io.Writer, header string, body string) {
 func printTicks(w io.Writer) {
 	fmt.Fprint(w, "```\n")
 }
+
+func writeHTML() {
```

Comment:
I would prefer that writeHTML took an input reader for the source data, or just a `string` or `[]byte`. It could also take a Writer as a parameter (similar to printMD). Passing in the file output path (the global `html` variable) is also ok, I mostly prefer not to use the global variables directly in utility functions.

Reply:
Fixed